PR: Add sqrtpi package (#507)
stdlib-js/stdlib: lib/node_modules/@stdlib/math/base/special/sqrtpi/README.md

<!--

@license Apache-2.0

Copyright (c) 2022 The Stdlib Authors.

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

   http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.

-->

# sqrtpi

> > Compute the positive [square root][square-root] of the product of pi and a provided positive number.

Reviewer: ```suggestion
> Compute the principal [square root][@stdlib/math/base/special/sqrt] of the product of 𝛑 and a positive number.
```

<section class="usage">

## Usage

```javascript
var sqrtpi = require( '@stdlib/math/base/special/sqrtpi' );
```

#### sqrtpi( x )

> Computes the positive [square root][square-root] of the product of pi and a provided positive number.

Reviewer: ```suggestion
Computes the principal [square root][@stdlib/math/base/special/sqrt] of the product of 𝛑 and a positive number.
```

We can refer to the documentation for `sqrt` within stdlib, rather than to an external source.

```javascript
var v = sqrtpi( 4.0 );

> // returns 3.5449

Reviewer: ```suggestion
// returns ~3.5449
```

v = sqrtpi( 10.0 );

> // returns 5.6049

Reviewer: ```suggestion
// returns ~5.6049
```

My guess is that these are approximate values, correct? Not exact? If the former, then we always include a `~` character to indicate that exact values may deviate from what is displayed.

v = sqrtpi( 0.0 );
// returns 0.0

v = sqrtpi( NaN );
// returns NaN
```

> For negative numbers, the positive [square root][square-root] of the product of pi and a provided positive number is **not** defined.

Reviewer: ```suggestion
For negative numbers, the principal [square root][@stdlib/math/base/special/sqrt] is **not** defined.
```

```javascript

> var v = sqrt( -4.0 );

Reviewer: ```suggestion
var v = sqrtpi( -4.0 );
```

<section class="related">

* * *

## See Also

-   <span class="package-name">[`@stdlib/math/base/special/sqrt`][@stdlib/math/base/special/sqrt]</span><span class="delimiter">: </span><span class="description">compute the principal square root of a double-precision floating-point number.</span>
-   <span class="package-name">[`@stdlib/math/base/special/rsqrt`][@stdlib/math/base/special/rsqrt]</span><span class="delimiter">: </span><span class="description">compute the reciprocal square root of a double-precision floating-point number.</span>

</section>

<!-- /.related -->

<!-- Section for all links. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->

<section class="links">

[square-root]: https://en.wikipedia.org/wiki/Square_root

<!-- <related-links> -->

[@stdlib/math/base/special/sqrt]: https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/math/base/special/sqrt

[@stdlib/math/base/special/rsqrt]: https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/math/base/special/rsqrt

> <!-- </related-links> -->

Reviewer: ```suggestion
[@stdlib/math/base/special/sqrt]: https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/math/base/special/sqrt

<!-- <related-links> -->

[@stdlib/math/base/special/rsqrt]: https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/math/base/special/rsqrt

<!-- </related-links> -->
```
